### ISSUE TITLE
bug: weather-based alarm does not show in overlay, disappears when toggled

### ISSUE THREAD

**Issue by rdj** [Supamiu/ffxiv-teamcraft#833]
**Describe the bug**
1. Weather-based alarms are not listed in the Overlay window. 
2. Weather-based alarms disappear from alarm groups when the on/off switch is toggled.

**To Reproduce**
1. Go to "Gathering Items"
2. Search for "Glimmerscale"
3. Click the Alarm + icon
4. Observe Glimmerscale has appeared in the alarm sidebar
5. Go to "Alarms"
6. Click the "Open Overlay" icon
7. Observe that Glimmerscale does not appear in the overlay (_bug 1_)
8. Drag the Glimmerscale alarm into an alarm group
9. If the alarm group is on, Glimmerscale is still in the alarm sidebar
10. Toggle the on/off switch for the alarm group
11. Observe that the Glimmerscale alarm has disappeared from the alarm group (_bug 2_)

**Expected behavior**
1. Glimmerscale should appear in the overlay window any time it is in the alarm sidebar.
2. Glimmerscale should not disappear from alarm groups unless deleted.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Desktop app
 - Version: v5.2.0

**Smartphone (please complete the following information):**

**Additional context**
Sorry for the two-part bug, but since weather-dependent alarms are a new feature in this version, it seemed ok to mention both problems in one issue.

**Comment by rdj** [comment]
Behavior persists in v5.2.1

**Comment by Supamiu** [comment]
Hey, thank you for the report :)

I'm going to investigate that tomorrow, hoping for a fix asap.

**Comment by rdj** [comment]
Cool. I'm obviously pretty excited about the feature! :)

**Comment by Supamiu** [comment]
Do you have any errors inside the console when getting this error? F12 on chrome or inside desktop app, hover your user name in top-right corner, open settings page and click the console button inside the desktop category.

**Comment by rdj** [comment]
Couple of not-super-relevant-looking errors when the app starts but nothing logs to the console when I add the alarm or toggle the switch. 

```
23:54:06.291 www.googletagmanager.com/gtag/js?id=UA-104948571-1 Failed to load resource: net::ERR_NAME_NOT_RESOLVED
23:54:06.630 main.4d7f7e97b8e86f865732.js:1 
Warning: This is a browser-targeted Firebase bundle but it appears it is being
run in a Node environment.  If running in a Node environment, make sure you
are using the bundle specified by the "main" field in package.json.

If you are using Webpack, you can specify "main" as the first item in
"resolve.mainFields":
https://webpack.js.org/configuration/resolve/#resolvemainfields

If using Rollup, use the rollup-plugin-node-resolve plugin and set "module"
to false and "main" to true:
https://github.com/rollup/rollup-plugin-node-resolve

plFh @ main.4d7f7e97b8e86f865732.js:1
23:54:09.849 thing.svg Failed to load resource: net::ERR_FILE_NOT_FOUND
```

**Comment by rdj** [comment]
Two more things. 

1. I noticed that the Glimmerscale alarm will disappear when any alarm group on/off switch is toggled, regardless of whether the Glimmerscale is in that alarm group, a different alarm group, or no alarm group. 

2. Tried the web site in Chrome 73.0.3683.86 and got the same behavior as the desktop app.

**Comment by Supamiu** [comment]
Are you logged in when you get this issue? Or is it as anonymous user?

**Comment by rdj** [comment]
Everything seems fixed in v5.2.2. Thanks for the quick turn around. 👍

**Comment by Supamiu** [comment]
Yay !
